easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.0.3rc3-GCC-9.2.0.eb

```diff
@@ -0,0 +1,23 @@
+name = 'OpenMPI'
+version = '4.0.3rc3'
+
+homepage = 'https://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '9.2.0'}
+
+source_urls = ['https://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['8c25d9cc4902f6ce6478675ed61076b2e399c0b148965fa6c2009f0425a9e4d9']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('hwloc', '2.1.0')
+]
+
+configopts = '--enable-mpi1-compatibility '
```

Comment:
Wouldn't it be better while we're still in the 2020a branch to remove this so we can detect what fails?

Reply:
good point, changed in 46631d2